puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -9,6 +9,8 @@
   $backend_glusterfs_name = 'glusterfs',
   $backend_iscsi          = false,
   $backend_iscsi_name     = 'iscsi',
+  $backend_netapp         = false,
```

Comment:
If I'm understanding correctly, I already have the changes for `pacemaker::cinder` in `puppet/modules/quickstack/manifests/pacemaker/cinder.pp`. (starting with the line this comment originated on)

To make changes in `quickstack::cinder`, do I want to add the parameters directly to `quickstack/manifests/cinder.pp`?  I don't see any other storage providers' code in there, but I can certainly do that if it makes sense.  I took another look through and found that I didn't add the parameters to `quickstack/manifests/storage_backend/cinder.pp` - would this solve the issue?